contracts/bridge/contracts/binance/LightClientBridge.sol

```diff
@@ -453,6 +469,16 @@ contract LightClientBridge is Bitfield, ValidatorRegistry, GuardRegistry {
 
         // Guard Registry double-check the commitmentHash
         checkGuardSignatures(commitmentHash, guardSignatures);
+
+        /**
+         * @notice If relayer submit correct commitmentHash in `newSignatureCommitment`, payback the collateral.
+         * else slash the collateral.
+         */
+        if (commitmentHash == data.commitmentHash) {
+            data.senderAddress.transfer(MIN_SUPPORT);
+        } else {
+            msg.sender.transfer(MIN_SUPPORT);
+        }
```

Comment:
`verifyCommitment` is called by `completeSignatureCommitment`, will the verify still work even if `commitmentHash != data.commitmentHash`?

This will cause `completeSignatureCommitment` finish unexpected, and still do `processPayload(commitment.payload, commitment.blockNumber);`